bundles/cis-k8s-1.5.1/README.md

```diff
@@ -1,12 +1,21 @@
-CIS Kubernetes Benchmark v.1.5.1
-==================================================
+# cis-k8s-1.5.1
 
 ## Description
+sample description
```

Comment:
We shouldn't be eliminating the description.

Reply:
Acked. That was mistake, I'll put it back in there.